nupic/research/frameworks/dendrites/modules/dendritic_mlp.py

```diff
@@ -51,6 +54,8 @@ class DendriticMLP(nn.Module):
     :param freeze_dendrites: whether to set `requires_grad=False` for all dendritic
                              weights so they don't train
     :param dendritic_layer_class: dendritic layer class to use for each hidden layer
+    :param output_nonlinearity: nonlinearity to apply to output. 'None' of no
```

Comment:
Perhaps be more specific and say `to apply to the final output layer`. At first I thought this was for the output of each layer (i.e. ReLU or K-winner).